rotkehlchen/api/rest.py

```diff
@@ -4469,9 +4469,16 @@ def export_history_events(
             headers.update({key: None for key in serialized_event})
 
         if directory_path is None:  # on download
-            with tempfile.TemporaryDirectory() as temp_dir:
+            with tempfile.TemporaryDirectory(ignore_cleanup_errors=True) as temp_dir:
```

Comment:
Delete the huge wall of text. This is a known issue

- https://www.scivision.dev/python-tempfile-permission-error-windows/
- https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory

Just add a link on the right as a comment along with probably noqa:501 to the first link

Reply:
We already do this in a few places. But not in all the places where the `tempfile.TemporaryDirectory` is used. You can grep about it.

Do you think this would mean that we may end up getting this bug in production?